src/vaadin-date-time-picker.html

```diff
@@ -21,15 +32,38 @@
       :host([hidden]) {
         display: none !important;
       }
+
+      .slot-container {
+        display: flex;
+      }
+
+      [part="date"],
+      .slot-container ::slotted([slot="date-picker"]) {
```

Comment:
This kind of selector is needed for shady CSS compliance (to work in Edge and IE11).

In native shadow DOM we wouldn't need the `.slot-container` wrapper element and could use CSS selector like `slot[name="date-picker"]::slotted(*)` instead, but that didn't work in Edge and IE11. See https://github.com/webcomponents/polyfills/tree/master/packages/shadycss#selector-scoping